src/test/java/org/json/junit/JSONObjectTest.java

```diff
@@ -3896,8 +3896,8 @@ public void issue743SerializationMapWith512Objects() {
 
     @Test
     public void issue743SerializationMapWith1000Objects() {
-      HashMap<String, Object> map  = buildNestedMap(1000);
-      JSONParserConfiguration parserConfiguration = new JSONParserConfiguration().withMaxNestingDepth(1000);
+      HashMap<String, Object> map  = buildNestedMap(500);
```

Comment:
Please add a comment explaining why the limit was changed. Also, method signature should be updated to '500Objects'.

Not why this testcase started failing in laptop tests.